oauth2/src/main/java/com/okta/spring/boot/oauth/OktaOAuth2AutoConfig.java

```diff
@@ -79,11 +79,12 @@ OAuth2UserService<OidcUserRequest, OidcUser> oidcUserService(
     static class OAuth2SecurityFilterChainConfiguration {
 
         @Bean
-        SecurityFilterChain oauth2SecurityFilterChain(HttpSecurity http, ClientRegistrationRepository clientRegistrationRepository) throws Exception {
+        SecurityFilterChain oauth2SecurityFilterChain(HttpSecurity http, ClientRegistrationRepository clientRegistrationRepository, OktaOAuth2Properties oktaOAuth2Properties) throws Exception {
             // as of Spring Security 5.4 the default chain uses oauth2Login OR a JWT resource server (NOT both)
             // this does the same as both defaults merged together (and provides the previous behavior)
             http.authorizeRequests((requests) -> requests.anyRequest().authenticated());
             Okta.configureOAuth2WithPkce(http, clientRegistrationRepository);
+            Okta.configureOAuth2WithAudience(http, clientRegistrationRepository, oktaOAuth2Properties);
```

Comment:
The audience overrides the PKCE parameters added on the previous line. Is there a way to chain these together so both parameters are added?